package.json

```diff
@@ -34,7 +34,8 @@
 		"test:unit": "jest",
 		"test:unit-watch": "jest --watch --coverage",
 		"test:integration": "node helpers/tester.js",
-		"test": "bolt test:format && bolt test:unit && bolt test:integration"
+		"test": "bolt test:format && bolt test:unit && bolt test:integration",
+		"create-component": "node ./helpers/create-component-cli.js"
```

Comment:
Why not map this to the `bolt new` script and move your code to `./helpers/cli.js`?

Reply:
Don't know how I missed that, will replace.